pkgs/top-level/all-packages.nix

```diff
@@ -31543,7 +31546,10 @@ with pkgs;
 
   ngt = callPackage ../development/libraries/ngt { };
 
-  nheko = libsForQt5.callPackage ../applications/networking/instant-messengers/nheko { };
+  nheko = libsForQt5.callPackage ../applications/networking/instant-messengers/nheko {
+    # https://github.com/NixOS/nixpkgs/issues/201254
+    stdenv = if stdenv.isLinux && stdenv.isAarch64 && stdenv.cc.isGNU then gcc11Stdenv else stdenv;
```

Comment:
I hope somebody remembers to remove this after Arm is upgraded to gcc11 but before nixpkgs upgrades to gcc12.  Otherwise, rebuild city...